oidc-controller/src/VCAuthn/Models/AuthSession.cs

```diff
@@ -22,19 +21,19 @@ public class AuthSession
         private string _requestParameters;
 
         [NotMapped]
-        public Dictionary<string,string> RequestParameters
+        public Dictionary<string, string> RequestParameters
         {
-            get => _requestParameters == null ? null : JsonConvert.DeserializeObject<Dictionary<string,string>>(_requestParameters);
+            get => _requestParameters == null ? null : JsonConvert.DeserializeObject<Dictionary<string, string>>(_requestParameters);
             set => _requestParameters = JsonConvert.SerializeObject(value);
         }
 
         private string _presentationRequest;
 
         [NotMapped]
-        public PresentationRequest PresentationRequest
+        public string PresentationRequest
```

Comment:
I'd probably just simplify this property now remove the `[NotMapped]` property, remove the `private string _presentationRequest;` And redefine this line to `public string PresentationRequest { get; set; }`